src/test/integration/integration-tests.py

```diff
@@ -0,0 +1,167 @@
+import utils
+import unittest
+import json
+import sys
+
+
+class TestStringMethods(unittest.TestCase):
+
+    # Test POST, GET, PUT, and DELETE /calendar/events
+    def test_events_combined(self):
+        # Test POST /calendar/events with valid body
+        server_id = utils.post_event(event_body)
+        validate_response(self, server_id, 202, "events")
+
+        valid_id = server_id.json()["data"]["id"]
+
+        # Test GET /calendar/events/{id} with valid ID
+        valid_get = utils.get_result_by_id("events", valid_id)
+        validate_response(self, valid_get, 200, "events")
+
+        # Test PUT /calendar/events/{id} with valid ID and body
+        valid_put = utils.put_event(valid_id, event_body)
+        validate_response(self, valid_put, 200, "events")
+
+        # Test DELETE /calendar/events/{id} with valid ID
+        valid_delete = utils.delete_event(valid_id)
+        validate_response(self, valid_delete, 204)
+
+    # Test GET /calendar/events
+    def test_get_events(self):
+        valid_get = utils.get_results("events")
+        validate_response(self, valid_get, 200)
+
+    # Test POST /calendar/events
+    def test_post_event(self):
+        # Invalid date in instance
+        good_date = event_body["data"]["attributes"]["instances"][0]["start"]
+        event_body["data"]["attributes"]["instances"][0]["start"] = "badDate"
+        bad_date = utils.post_event(event_body)
+        validate_response(self, bad_date, 400, message="Could not parse job")
```

Comment:
job?

Reply:
The message is referencing this https://github.com/osu-mist/osu-events-api/blob/develop/src/main/groovy/edu/oregonstate/mist/osuevents/resources/EventsResource.groovy#L175-L183. From what I understand, if there's an instance that has a start/end date that isn't in the right format, a 400 will get thrown with the "could not parse job object..." message. I could change the message to "Could not parse job object" to avoid any confusion though.